SharedSettings/Catalyst.MostRules.Error.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Catalyst.MostRules.Error</id>
+    <title>Catalyst shared Code Analisys Rules: Most rules as error</title>
+    <version>$version$</version>
+    <authors>Health Catalyst</authors>
+    <owners>Health Catalyst</owners>
+    <licenseUrl>https://github.com/HealthCatalyst/Fabric.SharedSettings</licenseUrl>
+    <projectUrl>https://github.com/HealthCatalyst/Fabric.SharedSettings</projectUrl>
+    <description>Deployment package for shared code analysis rules. This ruleset is based on Microsoft.AllRules.Errors.ruleset with some HealthCatalyst modifications.</description>
+  </metadata>
+  <files>
+    <file src="build\Catalyst.MostRules.Error.targets" target="build" />
```

Comment:
Looks like I'll need to do some tweaking to support this scenario -- I remember having discussed it, but apparently this method means that changes made to CA rules affect this file (instead of going into a custom, merge file).

Reply:
It looks like this method won't work to support that -- utilizing the .targets file, it runs these rules in addition to any custom ruleset. I can remove the .targets file from the pull request and we can still utilize the nuget packages for downloading the rulesets. It will require a little more work to consume them, but I'm not seeing a good way to consume this in a more automated fashion. @j2jensen

Reply:
That makes sense. Very unfortunate. But at least we can use the NuGet packages to keep the files up to date rather than having to go grab stuff from the repo from time to time. I think it also sets the stage for us to include the Roslyn Analyzers as dependencies, too, which will remove a few other manual steps and help us to ensure that the rulesets accommodate any known issues with the specific Roslyn Analyzers versions that we've tested.

Once you remove the targets, then I think I'll be ready to merge this pull request. I imagine we want to wait to update the Readme with instructions for the NuGet package, until after we've got the NuGet package published and tested, right?

Reply:
Yes, that's correct. Once we get the nuget package created, then we can update the Readme with how to consume it.